File: aula-09/exemplo.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
    char nome[100] = "Joao";
    int idade = 21;
    float valor = 34.82;

    printf("%s tem %d anos e possui R$ %.2f na carteira \n", nome, idade, valor);
}

File: aula-09/exemplo.py
# Alô, som!
nome = "Joao"
idade = 21
valor = 29.31

idade = int(input(f"Informe a idade de {nome}: "))

print(f"{nome} tem {idade} anos e possui R$ {valor} na carteira")
if idade < 21:
    print(f"{nome} ainda nao possui idade suficiente para maioridade civil.")
else:
    print(f"{nome} ja pode ser declarado civilmente capaz.")
print("Ate a proxima!")